src/components/Widgets/BaseWidget.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box } from '@mui/material';
+import { useRef } from 'react';
+import { GenericWidget } from './GenericWidget';
+import { BaseWidgetProps } from './widgetConfig';
+
+export const BaseWidget = ({ ...props }: BaseWidgetProps) => {
+  const containerRef = useRef<HTMLDivElement>(null);
+
+  return (
+    <GenericWidget {...props} ref={containerRef}>
+      <Box
+        sx={({ spacing }) => ({
+          height: '100%',
```

Comment:
same here
 ```css 
height : 1
``` 
would be more consistent.

Reply:
Same as above.